x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.test.ts

```diff
@@ -286,22 +286,15 @@ describe('endpoint list middleware', () => {
 
     it('should set ActivityLog state to loading', async () => {
       dispatchUserChangedUrl();
-      dispatchGetActivityLogLoading();
 
       const loadingDispatched = waitForAction('endpointDetailsActivityLogChanged', {
         validate(action) {
           return isLoadingResourceState(action.payload);
         },
       });
+      dispatchGetActivityLogLoading();
 
       const loadingDispatchedResponse = await loadingDispatched;
-      expect(mockedApis.responseProvider.activityLogResponse).toHaveBeenCalledWith({
```

Comment:
isn't this test checking the case when the user loads the page with the details=activity_log?

Reply:
Not specifially, no! It just tests that the `AsyncResourceState` helper correctly updates the `type` of `activityLog` in the redux store.

Reply:
There's a test for verifying data load using URL here `/endpoint_hosts/view/index.test.tsx#L880`(https://github.com/elastic/kibana/blob/265fed590e72d23ee918b07a8643ddf287f30e2f/x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx#L880)